.travis.yml

```diff
@@ -29,7 +23,7 @@ cache:
     - yarn
     - pip
     - directories:
-        - /home/travis/.pyenv/versions/
```

Comment:
Could we not add something like a before_cache phase to delete unwanted files/dependencies/logs?

Reply:
I don't think this is relevant to this PR